FirstProject/css/main.css

```diff
@@ -0,0 +1,90 @@
+header {
```

Comment:
I used this technique because in this particular situation I was very sure that my web page won't crush.(Because I used header tag just in a single page. If I need header tag in more pages I would use classes/ID's to style them)